core/conn/Makefile

```diff
@@ -25,16 +25,17 @@ include ../macros.gmk #top level
 
 RM      = /bin/rm
 ifeq ($(SQ_BUILD_TYPE),release)
-  CLIENT_TAR	?= ../../${DISTRIBUTION_DIR}/trafodion_clients-$(TRAFODION_VER).tgz
+  CLIENT_TAR	?= ../../${DISTRIBUTION_DIR}/apache-trafodion_clients-$(TRAFODION_VER)-incubating.tar.gz
```

Comment:
Instead of hard coding(apache_trafodion) can we use the environment variable TRAFODION_VER_PROD